libs/scripts/scripts.js

```diff
@@ -101,6 +101,8 @@ const locales = {
 };
 
 const config = {
+  geoRouting: 'off',
```

Comment:
This is weird. Why would we have geoRouting off, but fallbackRouting on? fallback routing doesn't even work without georouting.

Reply:
As you say it doesn't affect anything. I think if someone turns the geoRouting on, then we probably want fallback routing turned on by default as well. I agree it looks a bit weird though having something on when it has no initial effect.

Reply:
I've thought about it more, and it's probably not bad to be showing the modal "less" unless the devs decide to turn fallback routing on. I've changed it. 👍